centos.org/pipelines/lib/foremanDuffyJob.groovy

```diff
@@ -90,9 +90,7 @@ pipeline {
 
                                 duffy_scp_in('otel_env', 'otel_env', boxname, './')
                                 for(playbook in playbooks) {
```

Comment:
Thinking out loud: If we don't have a stage for it, does it still make sense to run it individually? I guess it's seen in the command so still some added value.

The reason I ask is that we can just run a single large pipeline and remove complexity.